doc/manuals/devel/sourceCode.md

```diff
@@ -448,10 +448,6 @@ When an entity is created or modified or when a subscription is created or modif
 [**mongoBackend**](#srclibmongobackend) decides when to notify and **ngsiNotify** executes the notification, with the help of the external library [libcurl](https://curl.haxx.se/libcurl/).
 Actually, a function from the [**rest** library](#srclibrest) is used: `httpRequestSend()`. Another important aspect of this library is that the notifications are sent by separate threads, using a thread pool if desired.
 
-We have subscriptions both for *context entities* and for *availability registrations* and thus, there are two types of notifications. Context entity subscriptions are by far more important (registration subscriptions are barely used at all) and the thread pool for notifications is used for context entity subscriptions only.
-
-### Context entities notifications
-
```

Comment:
An old leftover of context availability subscription which no longer makes sense in this documentation. I have taken the opportunity to clean it up.

NTC (informative)